openapi/paths/storefront/invoices@{id}@send-email.yaml

```
@@ -0,0 +1,27 @@
+parameters:
+  - $ref: ../../components/parameters/resourceId.yaml
+post:
+  x-products:
+    - Storefront
+  tags:
+    - Storefront invoices
+  summary: Resend an Invoice email
+  operationId: StorefrontPostInvoiceSendEmail
+  x-sdk-operation-name: post
+  security:
+    - CustomerJWT: []
+  description: Creates an email message to resend Invoice.
```

nitpick: capitalization 
```suggestion
  description: Creates an email message to resend a invoice.
```